### PR TITLE
udpate 20w13b 2 try

### PR DESCRIPTION
**Pull request by sbh1311**

### CODE REVIEW

**Comment by sbh1311** [comment]
i did a mistake

**Comment by sbh1311** [comment]
did it push the array too?

File: data/pc/20w13b/protocol.json [hunk 1]
```diff
@@ -3959,18 +3959,18 @@
...
-                    "0x47": "entity_equipment",
-                    "0x48": "experience",
-                    "0x49": "update_health",
-                    "0x4a": "scoreboard_objective",
-                    "0x4b": "set_passengers",
-                    "0x4c": "teams",
-                    "0x4d": "scoreboard_score",
-                    "0x4e": "spawn_position",
+                    "0x43": "spawn_position",
+                    "0x44": "display_scoreboard",
```
Comment (rom1504) [inline data/pc/20w13b/protocol.json]: you changed that name, change it back

File: data/pc/20w13b/protocol.json [hunk 2]
```diff
@@ -3959,18 +3959,18 @@
...
+                    "0x45": "entity_metadata",
+                    "0x46": "attach_entity",
+                    "0x47": "entity_velocity",
+                    "0x48": "entity_equipment",
+                    "0x49": "set_experience",
```
Comment (rom1504) [inline data/pc/20w13b/protocol.json]: you changed that name, change it back

File: data/pc/20w13b/protocol.json [hunk 3]
```diff
@@ -3959,18 +3959,18 @@
...
+                    "0x4a": "update_health", 
+                    "0x4b": "scoreboard_objective",
+                    "0x4c": "set_passengers", 
+                    "0x4d": "teams",
+                    "0x4e": "update_score",
```
Comment (rom1504) [inline data/pc/20w13b/protocol.json]: you changed that name, changed it back

**Pull request merged**